Package.swift

```diff
@@ -1,12 +1,21 @@
+// swift-tools-version:4.0
+
 import PackageDescription
 
 let package = Package(
-    name: "NetService",
-    targets: [
-        Target(name: "NetService", dependencies: ["Cifaddrs"])
-    ],
-    dependencies: [
-        .Package(url: "https://github.com/Bouke/DNS.git", majorVersion: 0, minor: 3),
-        .Package(url: "https://github.com/IBM-Swift/BlueSocket.git", majorVersion: 0, minor: 12)
-    ]
+  name: "NetService",
+  products: [
+    .library(name: "Cifaddrs", targets: ["Cifaddrs"]),
+    .library(name: "NetService", targets: ["NetService"]),
+  ],
+  dependencies: [
+    .package(url: "https://github.com/Bouke/DNS.git", .branch("master")),
```

Comment:
I've published a new version (1.1.0), so we don't need to target the master branch. Please specify that version `from: "1.1.0"` instead.